app/src/main/java/com/android/shelfLife/ui/utils/FoodItemDetails.kt

```diff
@@ -35,7 +35,6 @@ fun FoodItemDetails(foodItem: FoodItem) {
       modifier =
           Modifier.fillMaxWidth()
               .padding(horizontal = 16.dp, vertical = 8.dp)
-              .background(Color.White)
```

Comment:
Good job removing this hard coded colour. 
Alternatively, you also could have used `MaterialTheme.colorScheme.primaryContaine ` or some other themed colour, in order to keep the theme consistent throughout the app.